docs/source/models.rst

```diff
@@ -1646,29 +1646,32 @@ MLflow currently supports the following environment management tools to restore
 
 local
     Use the local environment. No extra tools are required.
-conda
-    Create environments using conda. Conda must be installed for this mode of environment reconstruction.
 
-    - `conda installation instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>`_
-virtualenv
+virtualenv (preferred)
     Create environments using virtualenv and pyenv (for python version management). Virtualenv and
     pyenv (for Linux and macOS) or pyenv-win (for Windows) must be installed for this mode of environment reconstruction.
 
     - `virtualenv installation instructions <https://virtualenv.pypa.io/en/latest/installation.html>`_
     - `pyenv installation instructions <https://github.com/pyenv/pyenv#installation>`_
     - `pyenv-win installation instructions <https://github.com/pyenv-win/pyenv-win#installation>`_
 
-    .. note::
-        Virtualenv support is still experimental and may be changed in a future MLflow release.
+conda
+    Create environments using conda. Conda must be installed for this mode of environment reconstruction.
+
+    .. warning::
+
+        By using conda, you're responsible for adhering to `Anaconda's terms of service <https://legal.anaconda.com/policies/en/?name=terms-of-service>`_.
```

Comment:
@dbczumar Added this warning to address [the comment above](https://github.com/mlflow/mlflow/pull/6507/files#r950651512).